### PR TITLE
Feature/user_model

### PR DESCRIPTION
**Pull request by ghost**
Users model created.

### CODE REVIEW

**Comment by ghost** [comment]
This should close issue# 3.

File: models/user.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class User extends Model {}
+
+// set up fields and rules for Users model
+Users.init(
```
Comment (HighDynamics) [inline models/user.js]: Should be `User.init`

**Review by HighDynamics — CHANGES_REQUESTED** [review]
Just the one change should do it.

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const { Model, DataTypes } = require("sequelize");
+const sequelize = require("../config/connection");
+
+class User extends Model {}
```
Comment (HighDynamics) [inline models/user.js]: In here is where the `checkPassword` method should go.

File: models/user.js (hunk 1 above)
Comment (ghost) [reply models/user.js]: Change made, see commit 91eca71

**Comment by ghost** [comment]
Added checkPassword method, if complete this will close issue# 3.

**Review by HighDynamics — APPROVED** [review]

**Pull request merged**